src/fiskaltrust.ifPOS/v1/de/Models/TseInfo.cs

```diff
@@ -6,28 +6,39 @@ namespace fiskaltrust.ifPOS.v1.de
     [DataContract]
     public class TseInfo
     {
-        [DataMember(Order = 1)]
+        [DataMember(Order = 10)]
         public long MaxNumberOfClients { get; set; }
 
-        [DataMember(Order = 2)]
+        [DataMember(Order = 20)]
         public long CurrentNumberOfClients { get; set; }
 
-        [DataMember(Order = 3)]
+        [DataMember(Order = 30)]
+        public IEnumerable<string> CurrentClientIds { get; set; }
+
+        [DataMember(Order = 40)]
         public long MaxNumberOfTransactions { get; set; }
 
-        [DataMember(Order = 4)]
+        [DataMember(Order = 50)]
         public long CurrentNumberOfTransactions { get; set; }
 
-        [DataMember(Order = 5)]
+        /*
+        [DataMember(Order = 60)]
         public IEnumerable<CertificateInfo> Certificates { get; set; }
 
-        [DataMember(Order = 6)]
+        [DataMember(Order = 70)]
         public IEnumerable<string> SerialNumbersBase64 { get; set; }
+        */
+
+        [DataMember(Order = 60)]
+        public string TsePublicKeyBase64 { get; set; }
```

Comment:
This should be a array, because there can be multiple public keys

Reply:
in my point of view, the tse-public-key is a single one and therefore also the tse-serial-number.
to be able to get multiple certificates out of it, maybe because of certificate-chain or maybe because of different usage for adminlog, syslog and transactionlog, i'll bring back a collection of certificates. (public IEnumerable<string> CertificatesBase64 { get; set; })

Reply:
I totally agree with that Tse Serialnumber stuff. I still would add another Enumerable which returns all serialnumbers (similar to the certificate one), since this is also one of the core functionalities of the secure element api.

Reply:
added 'public IEnumerable<string> SerialnumbersOctet { get; set; }' , naming not perfect but aligned to the other properties.